Documentation/ceph-cluster-crd.md

```diff
@@ -17,6 +17,8 @@ metadata:
   name: rook-ceph
   namespace: rook-ceph
 spec:
+  cephVersion:
+    image: ceph/ceph:v13.2.2-20181023
```

Comment:
Can this be changed now to a "better" looking tag or is this wanted?

Reply:
Using the full tag like this is recommended. Below in the cluster settings section I describe more about the reasoning. Maybe I should add a comment here so it's more obvious that they should find the latest tag.

Reply:
Mhh I just realized this is the CRD example and below are the settings, so it is fine.